app/controllers/queue_items_controller.rb

```diff
@@ -7,7 +7,7 @@ def index
   end
 
   def create
-    queue_item = QueueItem.new(queue_items_params)
+    queue_item = QueueItem.new(item_params)
```

Comment:
When you create Queue Items, you don't want user to have controller over it by permitting params. When you think about it - position will be calculated field, user is the current_user and video will pass an id so first you search whether that exists and whether that can be queued to the user. And then you do not use any params to build the new queue_item.